cli/src/test/java/org/opentosca/toscana/cli/testStrings.java

```diff
@@ -0,0 +1,302 @@
+package org.opentosca.toscana.cli;
+
+public class testStrings {
```

Comment:
I think usage of Strings is inappropriate here. The readability is quite bad. Instead, you could store every Response into its own textfile in the resource dir. Then, use:
`public static final String SYSTEM_STATUS = FileUtils.readFileToString("path/to/response");`